catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -544,6 +544,10 @@ private QueryRequest validateQueryRequest(QueryRequest queryRequest)
 
     int queryPageSize = queryRequest.getQuery().getPageSize();
 
+    if (queryPageSize > MAX_PAGE_SIZE || queryPageSize <= 0) {
+      LOGGER.error("IllegalArgumentException PAGE SIZE EXCEEDED");
```

Comment:
Should this if statement body also return queryRequest;?

Reply:
Yikes, that's not supposed to be there. Was testing something and I guess it got committed. I'll remove it